app/main.py

```diff
@@ -1,6 +1,32 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+import requests
+from dotenv import load_dotenv
+load_dotenv()
+
+
+BASE_URL = "http://api.weatherapi.com/v1/current.json?"
+API_KEY = os.environ.get("API_KEY")
+CITY = "Kiev"
+print(API_KEY)
+
+
+def get_weather(city: str = CITY, appi_key: str = API_KEY) -> print:
+
+    response = requests.get(BASE_URL + f"key={appi_key}&q={city}").json()
```

Comment:
use params argument for requests.get method here, it is bad practice to use f-strings for url

Reply:
Sorry I test some features and forgot to fix the requests city and print, also i fix mistake with response.